src/common/tests/test-mlib.c

```diff
@@ -362,13 +390,108 @@ _test_in_range (void)
    ASSERT (mlib_in_range (size_t, (size_t) ssize_max + 1u));
 }
 
+void
+_test_assert_aborts (void)
+{
+   int a = 0;
+   mlib_assert_aborts () {
+      a = 4;
+      abort ();
+   }
+   // Parent process is unaffected:
+   ASSERT (a == 0);
+}
+
+void
+_test_int_encoding (void)
+{
+   {
+      const char *buf = "\x01\x02\x03\x04";
+      const uint32_t val = mlib_read_u32le (buf);
+      mlib_check (val, eq, 0x04030201);
+   }
+
+   {
+      char buf[9] = {0};
+      char *o = mlib_write_i32le (buf, 0x01020304);
+      mlib_check (o, ptreq, buf + 4);
+      mlib_check (buf, streq, "\x04\x03\x02\x01");
+
+      o = mlib_write_i32le (o, 42);
+      mlib_check (o, ptreq, buf + 8);
+      mlib_check (buf, streq, "\x04\x03\x02\x01*");
+
+      o = mlib_write_i64le (buf, 0x0102030405060708);
+      mlib_check (o, ptreq, buf + 8);
+      mlib_check (buf, streq, "\x08\x07\x06\x05\x04\x03\x02\x01");
+   }
+}
+
+void
+_test_foreach (void)
```

Comment:
```suggestion
static void
_test_foreach (void)
```
Also applies to other new `_test_*` functions.